src/test/regress/expected/analyze.out

```diff
@@ -0,0 +1,38 @@
+/*
+Testing NULL counts 
+*/
+-- 1. empty table should have NO stats
+create table t (c int) distributed by (c);
+analyze t;
+select stanullfrac from pg_statistic where starelid = 't'::regclass::oid;
+ stanullfrac 
+-------------
+(0 rows)
+
+-- 2. table with ALL null values, expect to see 100% null values
+insert into t values (null);
+analyze t;
+select stanullfrac from pg_statistic where starelid = 't'::regclass::oid;
+ stanullfrac 
+-------------
+           1
+(1 row)
+
+-- 3. table with 1 null and 1 not-null values, expect to see 50% null values
+insert into t values (1);
+analyze t;
+select stanullfrac from pg_statistic where starelid = 't'::regclass::oid;
```

Comment:
another way to construct the test is to query the 'pg_stats' view. This is especially useful for MCV/MCF and histogram as the column definitions of pg_statistic are not very obvious (stavalues, stanumbers etc.).